spec/requests/api/pages_spec.rb

```diff
@@ -18,19 +18,30 @@ def json
 
   describe 'GET pages' do
     context 'with no specified language' do
-      let!(:featured_pages) { create_list :page, 5, featured: true }
-      let!(:mvp_pages) { create_list :page, 5, featured: false }
-      let!(:last_featured_page) { create :page, title: 'I am the latest featured page', featured: true, slug: 'garden_slug' }
-      let!(:last_mvp_page) { create :page, title: 'I am the latest test page', featured: false}
+      let!(:featured_pages) { create_list :page, 50, featured: true }
+      let!(:mvp_pages) { create_list :page, 50, featured: false }
```

Comment:
Creating 100 records is a bit much. For the integration specs it would be more valuable just checking that a request is met with a json response, and that the response has certain fields present.